### DISCUSSION TITLE
PyScript Autocomplete

### DISCUSSION THREAD

**Discussion by dmamelin** [custom-components/pyscript#556] [Show and tell]
I've developed an experimental tool for PyScript, focusing on IDE autocomplete functionality. This is my first Python project, and it involves mock objects and auto-generated classes for improved coding assistance. Feedback is appreciated.

I aimed to avoid interfering with PyScript's functionality, so some features are currently not implementable. 

https://github.com/dmamelin/pyscript_autocomplete
![image](https://github.com/custom-components/pyscript/assets/22717990/4820f2ed-c9b8-4372-b167-8c3bed38dc3e)

**Comment by ALERTua** [comment]
Great! However, I wish the installation and updating process was a tiny bit simpler. Of course, integrating into the pyscript itself would be ideal :)

**Reply by dmamelin** [reply]
Since the project is experimental and its future uncertain, I haven't yet focused on simplifying the installation or update process :)

Integrating this into PyScript can be easily implemented, but I'm interested in hearing @craigbarratt's opinion on the future development of PyScript
